app/services/item_servicer.rb

```diff
@@ -0,0 +1,15 @@
+class ItemServicer
+  def self.client
+    ItemClient
```

Comment:
It tends to be a little more conventional for a client to be instantiated versus using the class/class methods. Right now our apps are pretty simple. There is one request the uses a class, it completes its transaction and moves on. As apps get bigger and more complex there are scenarios where we might actually need to persist information in our client and/or have multiple instances at once which doesn't work well when we call directly on the class. This works but would be a less brittle if we use instances instead.